### PR TITLE
Fix tooltip

### PR DESCRIPTION
**Pull request by JackWilb**
Depends on #114 for the linter fixes.

This is a quick fix to re-implement the tooltip

### CODE REVIEW

File: src/components/NodeLink/NodeLink.vue [hunk 1]
```diff
@@ -204,13 +198,31 @@ export default {
...
+  background-color: white;
+
+  font-size:12.5px;
+  color:#000;
+  border-radius:5px;
+  padding:5px;
+  pointer-events:none;
+  -webkit-box-shadow:0 4px 8px 0 rgba(0,0,0,.2);
+  box-shadow:0 4px 8px 0 rgba(0,0,0,.2);
+  max-width:400px
```
Comment (waxlamp) [inline src/components/NodeLink/NodeLink.vue]: A space after the colon would make this more readable (like you have in lines 215-217 above).

Also, is the vendor version of `box-shadow` needed? Maybe @jtomeck could weigh in.

File: src/components/NodeLink/functionUpdateVis.ts [hunk 2]
```diff
@@ -205,7 +205,7 @@ export function showTooltip(message: string, event: MouseEvent, delay = 200) {
   tooltip.html(message)
     .style('left', `${event.clientX + 10}px`)
     .style('top', `${event.clientY - 20}px`);
-  tooltip.transition().duration(delay).style('opacity', 0.9);
+  tooltip.transition().duration(delay).style('opacity', 1);
```
Comment (waxlamp) [inline src/components/NodeLink/functionUpdateVis.ts:208]: Why change this? Seems like there's some value in remaining aware of the matrix view underneath the tooltip? It's ok to change it to opacity 1, but I was curious about your rationale.

**Review by waxlamp — DISMISSED** [review]
Looks great! Leaving this as an approval despite my questions; feel free to merge this as is or investigate those questions first.

File: src/components/NodeLink/functionUpdateVis.ts (hunk 2 above)
Comment (JackWilb) [reply src/components/NodeLink/functionUpdateVis.ts:208]: It just looked better without transparency, more readable

File: src/components/NodeLink/NodeLink.vue (hunk 1 above)
Comment (JackWilb) [reply src/components/NodeLink/NodeLink.vue]: Fixed your first point in b178885

**Review by waxlamp — APPROVED** [review]

**Pull request merged**